docs/04-kubernetes.md

```diff
@@ -1,41 +1,72 @@
 # Kubernetes
 
+## Requirements
+
+Go through the theory sources provided below (or find your own) to achieve the following requirements:
+
+-   You should unserstand the concept of container orchestration.
+-   You should understand the basic architecture of Kubernetes, inclduing the following components:
+    -   Master node/control plane
+    -   Worker node
+    -   etcd
+    -   kubelet
+    -   kube-proxy
+-   You should be able to manipulate the following Kubernetes resources:
+    -   Pods
+    -   ReplicaSets
+    -   Deployments
+    -   Services
+    -   Jobs
+    -   CronJobs
+-   You should be able to organise applications using labels and selectors.
+-   You can install `minikube`, start a local Kubernetes cluster, and use the `kubectl` command to iteract with it.
+-   You can explain the contents of a given Kubernetes manifest file, and you can apply it to a Kubernetes cluster.
+
 ## Theory
 
+### Articles
+
+TODO:
```

Comment:
If there are no articles, you can omit this subsection.

Reply:
I had one article, but hadn't got the time to put it here.